### ISSUE TITLE
Ported twisted/plugin.py to be Python3 compatible syntax

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7156]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7156|
|Type|enhancement|
|Created|2014-04-15 18:13:48Z|
|Branch|https://github.com/twisted/twisted/tree/fix-except-syntax-7156|

<details><summary>Searchable metadata</summary>

```
trac-id__7156 7156
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_fix_except_syntax_7156 branches/fix-except-syntax-7156
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1397585628000000 1397585628000000
changetime__1397590727000000 1397590727000000
version__None None
owner__Alex Alex

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

Patch available at https://github.com/twisted/twisted/pull/13/files

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

(In [42272]) Branching to fix-except-syntax-7156.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek commented|
|-|-|

(In [42273]) Apply Alex's patch

refs [#7156](https://github.com/twisted/twisted/issues/7156)

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set owner to Alex|
|-|-|

Looks good. I'll merge it.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_5"></a>|@radix set status to `closed`|
|-|-|

(In [42277]) Merge fix-except-syntax-7156: use py3 syntax in twisted/plugin.py

Author: Alex
Reviewer: radix
Fixes: [#7156](https://github.com/twisted/twisted/issues/7156)

Update twisted/plugin.py to use Python 3-compatible syntax.
